### PR TITLE
fix: Param helper hover getting cut off at bottom

### LINKED ISSUE #105422: Param helper hover getting cut off at bottom
Helper for `new SecretClient` from https://www.npmjs.com/package/@azure/keyvault-secrets

image

Looks like just the last line is cut off?

image

### PR DESCRIPTION
**Pull request by chenjigeng**
This PR fixes #105422

### CODE REVIEW

File: src/vs/editor/contrib/parameterHints/parameterHintsWidget.ts [hunk 1]
```diff
@@ -237,7 +241,11 @@ export class ParameterHintsWidget extends Disposable implements IContentWidget {
 		} else if (typeof signature.documentation === 'string') {
 			dom.append(this.domNodes.docs, $('p', {}, signature.documentation));
 		} else {
-			const renderedContents = this.renderDisposeables.add(this.markdownRenderer.render(signature.documentation));
+			const renderedContents = this.renderDisposeables.add(this.markdownRenderer.render(signature.documentation, {
+				asyncRenderCallback: () => {
+					this.domNodes?.scrollbar.scanDomNode();
+				}
+			}));
```
Comment (rheh) [inline src/vs/editor/contrib/parameterHints/parameterHintsWidget.ts:248]: This line and the lines surrounding lines are a duplicate of line 228-232.  Other than the dom element target (documentation or dom variable).  Should we refactor to bring these into the same place e.g.

```
		if (typeof signature.documentation === 'string') {
			dom.append(this.domNodes.docs, $('p', {}, signature.documentation));
		} else {
			const renderedContents = this.renderDisposeables.add(this.markdownRenderer.render(signature.documentation));
			const renderedContents = this.renderDisposeables.add(this.markdownRenderer.render(signature.documentation, {
				asyncRenderCallback: () => {
					this.domNodes?.scrollbar.scanDomNode();
				}
			}));
			renderedContents.element.classList.add('markdown-docs');
			const target = activeParameter?.documentation ?? dom;
			target.append(this.domNodes.docs, renderedContents.element);
```

Comment (chenjigeng) [reply src/vs/editor/contrib/parameterHints/parameterHintsWidget.ts:248]: just like this?
```typescript
private renderMarkdownDocs(markdown: IMarkdownString | undefined): IMarkdownRenderResult {
		const renderedContents = this.renderDisposeables.add(this.markdownRenderer.render(markdown, {
			asyncRenderCallback: () => {
				this.domNodes?.scrollbar.scanDomNode();
			}
		}));
		renderedContents.element.classList.add('markdown-docs');
		return renderedContents;
	}
```

**Review by rheh — APPROVED** [review]
LGTM

**Comment by chenjigeng** [comment]
@joaomoreno Can you help take a look at this PR

**Comment by mjbvz** [comment]
Thanks! Should be available in the first VS Code 1.53 insiders builds

**Pull request merged**